packages/webrtc/src/client.ts

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright © 2021 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import { ReqID, Incoming, Outgoing, Request, NotificationMethod, IncomingNotifications, OutgoingNotifications } from './api'
+
+function serialize (object: any): string {
+  return JSON.stringify(object)
+}
+
+function deserialize<D extends Incoming | Outgoing> (response: string): D {
+  return JSON.parse(response)
+}
+
+type Subscriber = (x: OutgoingNotifications) => Promise<void> | void
+
+export class Client {
+  private reqId: number = 0
+  private readonly pending = new Map<ReqID, (x: Outgoing) => void>()
```

Comment:
Is this stuff similar to `RequestProcessor`?

Reply:
Yep, as we discussed, it was simplified to match `webrtc-service` capabilities, I plan to migrate to `@anticrm/rpc` as soon as #55 be merged